Source/AwsCommonRuntimeKit/io/Stream.swift

```diff
@@ -11,12 +11,15 @@ private var vtable = aws_input_stream_vtable(seek: doSeek,
                                              destroy: doDestroy)
 //swiftlint:disable trailing_whitespace
 public class AwsInputStream {
+    //to be used in conversion from http request to sdk request and back during signing
+    public var byteBuffer: ByteBuffer
```

Comment:
not sure if this is the right more here. would love to discuss as I have no issue converting an `SdkHttpRequest` to an `HttpRequest` for CRT to send it but trying to convert it back from `HttpRequest` to `SdkHttpRequest` is difficult.